### ISSUE TITLE
app.py: virt-install --version check needs to be the same as in virtinstall.py

### ISSUE THREAD

**Issue by nacc** [cobbler/koan#8]
e.g.,

```
            # is libvirt new enough?
            rc, response, stderr_response = utils.subprocess_get_response(
                shlex.split('virt-install --version'), True, True)
            if rc == 0:
                if response:
                    version_str = response
                else:
                    version_str = stderr_response
            else:
                version_str = None
```

Right now, with older libvirt, version_str is ""

**Comment by brianjmurrell** [comment]
~sigh~ Sadly, yes.  This makes koan broken on distros such as EL 7.1

**Comment by jdobes** [comment]
@opoplawski @jmaas How is koan released nowadays? In koan git are not any releases, in cobbler git I see you released cobbler v2.6.11 in January. In the archive I see koan directory and koan section in specfile, however the koan sources are in some old version.

RH builds this koan for Fedora and EPEL - as a result this bug is still present even it's fixed in git since December.

**Comment by analogbyte** [comment]
@jdobes @opoplawski @jmaas I confirm this is stll in EPEL. Any news on a koan package update or even a release process?

**Comment by celesteking** [comment]
see attached

![EPEL package maintener](https://pbs.twimg.com/media/ClU6CMwUsAAJ3JU.jpg)

**Comment by opoplawski** [comment]
Please test https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2016-ed7d53c99a for EPEL7

**Comment by celesteking** [comment]
```
- looking for Cobbler at xx
- reading URL: xx
install_tree: xx
<type 'exceptions.AttributeError'>
'module' object has no attribute 'sub_process_get_response'
  File "/usr/lib/python2.7/site-packages/koan/app.py", line 274, in main
    k.run()
   File "/usr/lib/python2.7/site-packages/koan/app.py", line 424, in run
    self.virt()
   File "/usr/lib/python2.7/site-packages/koan/app.py", line 847, in virt
    return self.net_install(after_download)
   File "/usr/lib/python2.7/site-packages/koan/app.py", line 634, in net_install
    rc, response, stderr_response = utils.sub_process_get_response(shlex.split('/usr/bin/virt-install --version'), True, True)
```

**Comment by opoplawski** [comment]
Crap.  Updated again.

**Comment by analogbyte** [comment]
This issue solved when using koan-2.6.11-6.git95749a6.el7.noarch.rpm. Koan generally works for me, but I haven't had the chance to test all features and I did not test the new cobbler-* package. Thank you.

**Comment by SchoolGuy** [comment]
I think as @debugloop described this can be closed for now. If we have new problems with newer versions of cobbler/koan we should create a separate issue for these failures.
